packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -175,6 +173,31 @@ export interface IGCStats {
 	updatedAttachmentBlobCount: number;
 }
 
+/**
+ * The statistics of the system state after a garbage collection sweep phase run.
+ * @public
+ */
+export interface ISweepPhaseStats {
+	/** The number of nodes in the lifetime of the container. */
+	lifetimeNodeCount: number;
```

Comment:
Does this also include DDSes, or is it just the sum of the other two?

Reply:
It includes DDS too